docs/detections/rules-ui-create.asciidoc

```diff
@@ -517,12 +520,12 @@ minutes at 14:03 but it does not run until 14:04, it will run again at 14:09.
 [[rule-action-variables]]
 ==== Alert notification placeholders
 
-You can use http://mustache.github.io/[mustache syntax] to add the following placeholders to <<rule-notifications, rule action>> fields:
+You can use http://mustache.github.io/[mustache syntax] to add variables to notification messages. The action frequency you choose determines the variables you can select from.   
+
+The following variables can be passed if the rule’s action frequency is a summary of alerts or per alert: 
```

Comment:
When I first read this, I didn't understand why it was phrased this way (you can do X if Y is true). A rule's action frequency is _always_ either **Summary of alerts** or **For each alert**, because those are the only two options, and so these variables will _always_ be available.

I had to read further to see that some other variables are only available for certain frequency settings, but I still don't think this statement needs to be phrased as a conditional, when the condition will always be true.

```suggestion
The following variables can be passed for all rules: 
```